task_7/task_7_1.js

```diff
@@ -0,0 +1,26 @@
+const fs = require("node:fs/promises");
+const path = require("node:path");
+
+const startPath = __dirname;
+
+async function listDirsAndFiles(dir) {
+  try {
+    const files = await fs.readdir(dir, { withFileTypes: true });
+    for (const file of files) {
+      const currentFile = path.join(dir, file.name);
+      if (file.isDirectory()) {
```

Comment:
As a another option you can use fs.cp for copying both files and directories.

Reply:
Done!